README.docker.md

```diff
@@ -0,0 +1,26 @@
+# Running in Docker
+
+## Setup
+
+Install docker and docker-compose
+
+Then setup your environment variables:
+
+    sed 's/export //' env.test > env.docker
+
+## Running
+
+    docker-compose build
```

Comment:
@jonocodes from bash these instuctions are failing for me.
        
         MacBook-Pro:PaperBadger rcpeters$ sed 's/export //' env.test > env.docker
         MacBook-Pro:PaperBadger rcpeters$ docker-compose build
         mongo uses an image, skipping
         Building paperbadger
         ERROR: Couldn't connect to Docker daemon - you might need to run          `docker-machine start default`.
         docker-machine start default
         Machine "default" is already running.